module-1/README.md

```diff
@@ -63,55 +117,255 @@ export const environment = {
-    * Provide a [globally unique name](https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html) for your hosting bucket
-* Run `amplify publish`
+#### Module 1 Files
+
+Next, Within the lib folder you should find a file with the default name of `cdk-stack.ts`.  Rename this file to `Module1Stack.ts`.
+
+Open the file `Module1Stack.ts` in VS Code and ename the class to `Module1Stack`, as illustrated below.
+
+```typescript
+import cdk = require('@aws-cdk/cdk');
+
+export class Module1Stack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    // The code that defines your stack goes here
+  }
+}
+```
+
+Following the change of filename and classname, you should now update the references in the `cdk.ts` file, as such.
+
+```typescript
+#!/usr/bin/env node
+import 'source-map-support/register';
+import cdk = require('@aws-cdk/cdk');
+import { Module1Stack } from '../lib/Module1Stack';
+
+const app = new cdk.App();
+new Module1Stack(app, 'Module1');
+```
+
+Now let's go through defining the S3 and CloudFront infrastructure.
+
+#### Define the S3 bucket
+
+First of all, we are going to define our S3 bucket and define the web index document as 'index.html'
+
+```typescript
+const projectName = "mythical-mysfits";
+const bucket = new s3.Bucket(this, "Bucket", {
+  bucketName: projectName + "-frontend-" + cdk.Aws.accountId,
+  websiteIndexDocument: "index.html"
+});
+```
+
+#### Restricted access to the S3 bucket
+
+Next we obtain the cloudfront origin access identity so that the s3 bucket may be restricted to it.  Then we define an IAM policy to restrict access to the S3 bucket so that only CloudFront distribution can access it.
+
+```typescript
+const origin = new cloudfront.CfnCloudFrontOriginAccessIdentity(this, "BucketOrigin", {
+  cloudFrontOriginAccessIdentityConfig: {
+    comment: projectName
+  }
+});
+
+const bucketPolicy = new iam.PolicyStatement()
+  .addPrincipal(new iam.CanonicalUserPrincipal(
+    origin.cloudFrontOriginAccessIdentityS3CanonicalUserId
+  ))
+  .addAction("s3:GetObject")
+  .addResource(bucket.bucketArn)
+  .addResource(bucket.bucketArn + "/*");
+bucket.addToResourcePolicy(bucketPolicy);
+```
+
+#### CloudFront Distribution
+
+Next, Write the definition for a new CloudFront web distribution.
+
+```typescript
+const cdn = new cloudfront.CloudFrontWebDistribution(this, "CloudFront", {
+  viewerProtocolPolicy: cloudfront.ViewerProtocolPolicy.AllowAll,
+  priceClass: cloudfront.PriceClass.PriceClassAll,
+  originConfigs: [
+    {
+      behaviors: [
+        {
+          isDefaultBehavior: true,
+          maxTtlSeconds: undefined,
+          allowedMethods:
+            cloudfront.CloudFrontAllowedMethods.GET_HEAD_OPTIONS
+        }
+      ],
+      originPath: `/web`,
+      s3OriginSource: {
+        s3BucketSource: bucket,
+        originAccessIdentityId: origin.cloudFrontOriginAccessIdentityId
+      }
+    }
+  ]
+});
+```
+
```

Comment:
I think we should use the [s3-deployment module](https://docs.aws.amazon.com/cdk/api/latest/docs/aws-s3-deployment-readme.html)

Reply:
Can you update this against the new file.  I don't see what you are referring to